static/sass/styles.scss

```diff
@@ -678,5 +685,65 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
 }
 
 .is-ticked.is-play-button::before {
-  @include vf-icon-video-play($color-mid-dark);
+  @include vf-icon-video-play($color-mid-dark); 
+}
+
+// XXX: pad content so it doesn't get to close to text in the adjacent column. Idea borrowed from the guardian. Propose this to vanilla.
+.u-pad--right {
+  padding-right: $sph--x-large;
+}
+
+.u-align-social-icons-to-text {
+  vertical-align: text-bottom;
 }
+
+.l-article {
```

Comment:
Layouts should be the initial elements of the page. In this case you are only wrapping the `main`. This is confusing as it's a different construct to all others layouts.

Reply:
good point, I've renamed refactored that whole part. we have no access to the main here, so I've called it p-article to be clearer. I've also put the grid functionality into component classes so no nesting.